lib/components/T3MediaFile/type/Audio.vue

```diff
@@ -1,17 +1,15 @@
-<template functional>
+<template>
   <audio
-    v-bind="data.attrs"
-    :class="[data.class, data.staticClass]"
+    v-bind="$attrs"
```

Comment:
is there a need to use v-bind="$attrs" since we don't use inheritAttrs:false here ?